packages/e2e-test-utils-playwright/src/metrics/index.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * External dependencies
+ */
+import lighthouse from 'lighthouse';
+import type { Page } from '@playwright/test';
+
+export class Metrics {
```

Comment:
Note to self: since fixtures can access `testInfo`, we could provide a `saveResults()` method here. For example:

```ts
async saveResults( results: Record< string, unknown > ) {
	await this.testInfo.attach( 'results', {
		body: JSON.stringify( results, null, 2 ),
		contentType: 'application/json',
	} );
}
```

Or even a single `collectMetrics()` method that does everything from collection to attaching results.